config/routes.rb

```diff
@@ -8,8 +8,8 @@
   resources :users do
     resources :posts do
       member do
-        post 'like'
-        delete 'unlike'
+        post 'like'  # Change this line to a post request
+        delete 'unlike' # Change this line to a delete request
       end
       resources :comments
```

Comment:
- [ ] [OPTIONAL] 
Let's be more judicious by allowing only required routes in when using resources for the _post_ and _comment_ routes, so that unrequired routes are not exposed.